client/src/components/QnAcomponents/QuestionsListItem.jsx

```diff
@@ -1,85 +1,106 @@
 import React from 'react';
 import QuestionsListItemAnswer from './QuestionsListItemAnswer.jsx';
 import AddAnswerForm from './AddAnswerForm.jsx';
+import axios from 'axios';
 
 class QuestionsListItem extends React.Component {
-  constructor(props){
+  constructor(props) {
     super(props);
     this.state = {
       isAddAnswerClicked: false,
       answers: {},
       isMoreAnswersShown: false
-    }
+    };
     this.clickOnMoreAnswers = this.clickOnMoreAnswers.bind(this);
     this.addAnswerHandleClick = this.addAnswerHandleClick.bind(this);
+    this.clickOnHelpful = this.clickOnHelpful.bind(this);
   }
 
-  componentDidMount(){
+  componentDidMount() {
+
     let answersToShow = Object.values(this.props.question.answers);
-    if(answersToShow.length > 2){
+    if (answersToShow.length > 2) {
       this.setState({
         isMoreAnswersShown: true
-      })
+      });
     }
     answersToShow = answersToShow.slice(0, 2);
     this.setState({
       answers: answersToShow
-    })
+    });
   }
 
-  clickOnMoreAnswers(){
+  clickOnMoreAnswers() {
```

Comment:
clickOnMoreAnswers, addAnswerHandleClick, and clickOnHelpful seem to not be consistent in your naming scheme

Maybe more like 'handleClick_MoreAnswers' or something more universal for all your click handlers?